test/check-machines-snapshots

```diff
@@ -30,7 +30,7 @@ sys.path.append(os.path.join(os.path.dirname(TEST_DIR), "bots/machine"))
 from machineslib import VirtualMachinesCase  # noqa
 from testlib import nondestructive, test_main  # noqa
 
```

Comment:
Can you please add this comment here,

libvirt-dbus nnapshot APIs are available since 1.4.0 https://github.com/libvirt/libvirt-dbus/commit/642b1b71


for further reference